datasets/_wikidata/peps/crawler.py

```diff
@@ -64,7 +64,9 @@ def crawl_holder(
         # Avoid constructing a proxy which emits a warning
         # before we discard it.
         return
-    entity.add("birthDate", birth_date)
+    if birth_date is not None and birth_date[:10].endswith("-01-01"):
+        birth_date = birth_date[:4]
+    entity.add("birthDate", birth_date, original_value=holder.get("person_birth"))
```

Comment:
so, there is `?timenode wikibase:timePrecision     ?timeprecision.` but we're querying 7 dates in the query where this comes from, so for what we'd gain I'd say this is a fair simplification

Reply:
The other problem is humans: a ton of people in Wikidata actually have `-01-01` entered as their date, although it is wrong... so the precision thing is also just used wrong.